ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -4552,7 +4553,7 @@ private static Path getManifestDir(Path specPath, long writeId, int stmtId, Stri
     if (isDelete) {
       deltaDir = AcidUtils.deleteDeltaSubdir(writeId, writeId, stmtId);
     }
-    Path manifestPath = new Path(manifestRoot, "_tmp." + deltaDir);
+    Path manifestPath = new Path(manifestRoot, Utilities.toTempPath(deltaDir));
```

Comment:
`Utilities.toTempPath` is used in multiple places, would it be possible to isolate this change just for CTAS?

Reply:
@deniskuzZ I dont think it is a good idea to isolate specifically for CTAS since it is embedded within FileSinkOperator. I would keep this directory generic for all write operations.
Even if we do it specifically, then we have to introduce more logic for handling CTAS cases and this might not be a fault tolerant approach.